### PR TITLE
Removes a bad word from a poster

### PR DESCRIPTION
**Pull request by ClownMoff**
## About The Pull Request

There is a big no-no word on the no ERP poster that I replaced with another one

## Why It's Good For The Game
It's a very violent word. I once referenced the description of the poster in game and it felt pretty violent. Overall it's not a nice thing to be reading while playing

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

imagen

</details>

## Changelog
:cl: Varo
tweak: change a big no-no word in a poster
/:cl:

### CODE REVIEW

**Comment by St0rmC4st3r** [comment]
Um, but rape is bad and we are against it? The action is bad and violent I completely agree, but I don't see a problem with a poster that says "Rape is bad".
Let us not start an euphemism and moral treadmill on this one.

**Comment by ClownMoff** [comment]
I don't think people would roleplay rape anyways, and while i'm usually on the side  of just telling people to not be so soft with certain things, I think this is such a sensitive topic that shouldn't be mentioned that slightly. Maybe I'm just being a snowflake who knows ¯\_(ツ)_/¯

**Comment by TsunamiAnt** [comment]
no you are right, the vibes are bad on this. I like this pr.

File: code/game/objects/effects/contraband.dm [hunk 1]
```diff
@@ -603,7 +603,7 @@
 
 /obj/structure/sign/poster/official/no_erp
 	name = "No ERP"
-	desc = "This poster reminds the crew that Eroticism, Rape and Pornography are banned on Nanotrasen stations."
+	desc = "This poster reminds the crew that Eroticism, Rubbing and Pornography are banned on Nanotrasen stations."
```
Comment (DrDuckedGoose) [inline code/game/objects/effects/contraband.dm]:
```suggestion
	desc = "This poster reminds the crew that Eroticism, Rape and Pornography are banned on Nanotrasen stations."
```

**Comment by Rukofamicom** [comment]
This is not necessary to change. 

"Rape is bad" is not a message that needs censoring.

**Comment by JixS4v** [comment]
> This is not necessary to change.
> 
> "Rape is bad" is not a message that needs censoring.

I think the bad part about this is conflating ERP with Rape

**Comment by MarkusLarsson421** [comment]
Perhaps a simple change to "r*pe" to hint towards a player reading this that you shouldn't say it. Seeing as I recall this PR being made in response to Varo using said word in-character if I am not mistaken. Had there been an asterisk there, he might've understood it is not an excuse to say it in-character.

**Comment by ClownMoff** [comment]
Yeah I did a treasure hunt and I wanted to reference dorms and I used the No ERP sign description and it turned out kinda violent. Soo-Min didn't appreciate it a lot

File: code/game/objects/effects/contraband.dm (hunk 1 above)
Comment (ToasterBan) [reply code/game/objects/effects/contraband.dm]: rubbing is a weird word, could just have no word there and have the R represent the second letter of eroticism

Comment (PowerfulBacon) [inline code/game/objects/effects/contraband.dm]:
```suggestion
	desc = "This poster reminds the crew that Eroticism, Reproduction and Pornography are banned on Nanotrasen stations."
```

File: _maps/map_files/FlandStation/FlandStation.dmm [hunk 2]
```diff
@@ -97298,7 +97298,7 @@
 	},
 /obj/structure/closet/wardrobe/grey,
 /obj/structure/sign/poster/official/no_erp{
-	desc = "This poster reminds the crew that Eroticism, Rape and Pornography are banned on Nanotrasen stations. You feel as if this poster reminds you of an event that happened here... and that it's a placeholder for another poster?";
+	desc = "This poster reminds the crew that Eroticism, Rubbing and Pornography are banned on Nanotrasen stations. You feel as if this poster reminds you of an event that happened here... and that it's a placeholder for another poster?";
```
Comment (PowerfulBacon) [inline _maps/map_files/FlandStation/FlandStation.dmm:97301]:
```suggestion
	desc = "This poster reminds the crew that Eroticism, Reproduction and Pornography are banned on Nanotrasen stations. You feel as if this poster reminds you of an event that happened here... and that it's a placeholder for another poster?";
```

Comment (PowerfulBacon) [inline _maps/map_files/FlandStation/FlandStation.dmm:97301]:
```suggestion
	desc = "This poster reminds the crew that Eroticism, Reproduction and Pornography are banned on Nanotrasen stations. You feel as if this poster reminds you of an event that happened here... and that it's a placeholder for another poster?";
```

Comment (PowerfulBacon) [inline _maps/map_files/FlandStation/FlandStation.dmm]:
```suggestion
	desc = "This poster reminds the crew that Eroticism, Reproduction and Pornography are banned on Nanotrasen stations. You feel as if this poster reminds you of an event that happened here... and that it's a placeholder for another poster?";
```

**Comment by PowerfulBacon** [comment]
I can't merge the second change for some reason to merge

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**